R/utils.R

```diff
@@ -689,3 +689,41 @@ try_redcapr <- function(expr, call = caller_env()) {
 db_has_repeat_forms <- function(db_data) {
   "redcap_repeat_instance" %in% names(db_data)
 }
+
+#' @title
+#' Remove rows with empty data
+#'
+#' @description
+#' Check for empty rows in columns that aren't necessary for composite key
```

Comment:
I think the idea is that we want to remove rows that are empty in all [data columns](https://chop-cgtinformatics.github.io/REDCapTidieR/articles/glossary.html#data-column). These include columns necessary for composite key spec (i.e. [identifier columns](https://chop-cgtinformatics.github.io/REDCapTidieR/articles/glossary.html#identifier-column)) but also the survey and and form completion columns.

Reply:
Sounds good, changing the first sentence to `"Remove rows that are empty in all associated data columns (those derived from fields in REDCap)."`